README.md

```diff
@@ -39,3 +39,36 @@ Just follow these simple steps to enable touch events in your jQuery UI app:
     ```
 
 _Tested on iPad, iPhone, Android and other touch-enabled mobile devices._
+
+## Using Touch Punch with RequireJS
+
+jQuery UI 1.11+ is required for using Touch Punch with RequireJS.
+
+### Configuration
+
+The base path of jQuery UI have to be configured in the require config json:
+
+```javascript
+requirejs.config({
+    paths: {
+        // for example with bower
+       'jquery-ui': 'bower_components/jquery-ui',
```

Comment:
My example was not exact (writing from memory). its `bower_components/jquery-ui/ui`